source/quickstart/container.rst

```diff
@@ -6,35 +6,353 @@ Quickstart for Containers
+            - The ``docker run`` command starts the MinIO container with the ``-v`` argument to map the local path (``~/minio/data``) to the specified virtual container directory (``/data``).
+            - ``-p`` binds a local port to a container port.
+            - ``-v`` sets a file path as a persistent volume location for the container to use.'
+            - ``-e`` sets the environment variables ``MINIO_ROOT_USER`` and ``MINIO_ROOT_PASSWORD``, respectively.
+              Change the example values to use for your container.
+            - When MinIO writes data to ``/data``, that data actually writes to the local path ``~/minio/data`` where it can persist between container restarts.
+              You can replace ``${HOME}/minio/data`` with another location in the user's home directory to which the user has read, write, and delete access.
+                 
+         .. tab-item:: Windows
+   
+            .. code-block:: shell
+               :class: copyable
+   
+               docker run \
+                  -p 9000:9000 \
+                  -p 9001:9001 \
+                  --name minio1 \
+                  -e "MINIO_ROOT_USER=ROOTUSER" \
+                  -e "MINIO_ROOT_PASSWORD=CHANGEME123" \
+                  -v D:\data:/data \
+                  quay.io/minio/minio server /data --console-address ":9001"
+   
+            The example above works this way:
+   
+            - The ``docker run`` command starts the MinIO container with the ``-v`` argument to map the local path (``D:\data``) to the specified virtual container directory (``/data``).
+             - ``-p`` binds a local port to a container port.
+            - ``-v`` sets a file path as a persistent volume location for the container to use.'
+            - ``-e`` sets the environment variables ``MINIO_ROOT_USER`` and ``MINIO_ROOT_PASSWORD``, respectively.
+              Change the example values to use for your container.
+           - When MinIO writes data to ``/data``, that data actually writes to the local path ``D:\data`` where it can persist between container restarts.
+              You can replace ``D:\data`` with another local file location to which the user has read, write, and delete access.
 
-   Contents of Step 2
+#. Connect your browser
 
-#. Step 3
+   Open http://127.0.0.1:9000 in a web browser to access the :ref:`MinIO Console <minio-console>`. 
+   Alternatively, you can enter any of the network addresses specified as part of the container command output.
 
-   Contents of Step 3
+   Access the :ref:`minio-console` by going to a browser and enter one of the Console address specified in the container command's output.
+   For example, :guilabel:`Console: http://192.0.2.10:9001 http://127.0.0.1:9001` in the example output indicates two possible addresses to use for connecting to the Console.
+
+   If you go to port ``9000``, which is used for connecting to the API, you are automatically redirected to the Console port.
```

Comment:
```suggestion
   If you go to port ``9000``, which is used for connecting to the API, MinIO automatically redirects to the Console port.
```

Removing passive

Reply:
Also maybe move this up as part of 196 and simplify as "MinIO automatically redirects browser access from the API port (``9000``) to the Console port.

Reply:
Moved up.